.github/workflows/java-cd.yml

```diff
@@ -135,8 +149,15 @@ jobs:
             - name: Build java client
               working-directory: java
               run: |
-                  ./gradlew --build-cache :client:publishToMavenLocal -Psigning.secretKeyRingFile=secring.gpg \
-                  -Psigning.password="${{ secrets.GPG_PASSWORD }}" -Psigning.keyId=${{ secrets.GPG_KEY_ID }}
+                  if [[ "${{ matrix.host.build_type }}" == "musl" ]]; then
+                    rustup target add ${{ matrix.host.TARGET }}
```

Comment:
You can move these 2 lines to another step. Please use yml if:
```
if: ${{ ... }}
run: ...
```

Probably this should be in install rust action. Isn't it there already? How build for node client works?

Reply:
Moved into its own action. The install rust action itself doesn't handle this case properly if this isn't done. I'm not sure why the node client seems to work without it, but with a similar setup the java build fails.